src/components/AssetSelection/helpers.ts

```diff
@@ -0,0 +1,30 @@
+import type { ButtonProps } from '@chakra-ui/react'
+
+const disabledStyle = { cursor: 'not-allowed' }
+
+export const getStyledMenuButtonProps = ({
```

Comment:
same q. here

Reply:
This helper is to get identical styling props to be shared across both component types without duplication